discord/interactions.py

```diff
@@ -675,13 +837,22 @@ async def defer(self, *, ephemeral: bool = False, thinking: bool = False) -> Non
             In UI terms, this is represented as if the bot is thinking of a response. It is your responsibility to
             eventually send a followup message via :attr:`Interaction.followup` to make this thinking state go away.
             Application commands (AKA Slash commands) cannot use :attr:`InteractionResponseType.deferred_message_update`.
+        with_response: :class:`bool`
+            Whether to return the interaction response callback resource.
+
+            .. versionadded:: 2.5
 
         Raises
         -------
         HTTPException
             Deferring the interaction failed.
         InteractionResponded
             This interaction has already been responded to before.
+
+        Returns
+        -------
+        Optional[:class:`InteractionCallback`]
+            The interaction callback resource, or ``None``.
```

Comment:
```suggestion
            The interaction callback resource.
```

As `Optional[T]` implies "or None" it does not need to be outright stated.

Reply:
You are indeed correct, my apologies.
Happy to disregard these comments in this case.